Source/WebKit/UIProcess/Cocoa/WKTextExtractionItem.swift

```diff
@@ -34,8 +33,7 @@ import Foundation
     }
 }
 
-@available(iOS 17.0, macOS 12.0, *)
-@objc public enum WKTextExtractionContainer: Int {
+@objc enum WKTextExtractionContainer: Int {
```

Comment:
`@objc` doesn't actually apply to an enum, because it isn't an Objective-C type (just a C one). You should be able to delete this and just use the `NS_ENUM` defined enum directly, since this is actually just shadowing the type. It's possible that isn't exposed in a module (I notice that `WKTextExtrationItem.h` has Project visibility), which would need to be resolved using either an SPI module or by promoting that header to Private, if possible. Doing this just worries me because it means the two enums could get out of sync.